### PR TITLE
Web Package Methods Error Added

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Added throw error to web-mode methods so they fail on UI as well if there occur an error

### CODE REVIEW

File: packages/web/src/lib/myalgowallet-mode.ts [hunk 1]
```diff
@@ -85,6 +85,7 @@ export class MyAlgoWalletSession {
 			}
 		} catch (err) {
 			error(err);
+			throw err
```
Comment (rmeena840) [inline packages/web/src/lib/myalgowallet-mode.ts:88]: What does `error(err)` does then?

File: packages/web/src/lib/myalgowallet-mode.ts [hunk 2]
```diff
@@ -148,72 +162,90 @@ export class MyAlgoWalletSession {
...
+			const toBeSignedTxns: TransactionInGroup[] = execParams.map(
+				(txn: ExecParams, index: number) => {
+					return txn.sign === SignType.LogicSignature
+						? { txn: txns[index], shouldSign: false } // logic signature
+						: { txn: txns[index], shouldSign: true }; // to be signed
+				}
+			);
+			// only shouldSign txn are to be signed, algowallet doesn't accept lsig ones
+			const nonLsigTxn = toBeSignedTxns.filter((txn) => txn.shouldSign);
+			if (nonLsigTxn.length > 0) {
```
Comment (rmeena840) [inline packages/web/src/lib/myalgowallet-mode.ts:225]:
```suggestion
			if (nonLsigTxn.length) {
```

**Review by rmeena840 — COMMENTED** [review]
Left comment

File: packages/web/src/lib/myalgowallet-mode.ts (hunk 1 above)
Comment (Megha-Dev-19) [reply packages/web/src/lib/myalgowallet-mode.ts:88]: it logs the error (`error` is our internal logger)

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**